docs/setup/storage/s3.md

```diff
@@ -11,49 +11,115 @@ has_children: false
+[`lakectl`](../../reference#lakectl-fs), the [Spark metadata client](../../reference/spark-client.md),
+or the [S3 gateway](../../integrations/spark.md#access-lakefs-using-the-s3a-gateway).
+
+```json
+{
+   "Id": "<POLICY_ID>",
+   "Version": "2012-10-17",
+   "Statement": [
+      {
+         "Sid": "<STMT_ID1>",
          "Action": [
             "s3:GetObject",
-            "s3:GetObjectVersion",
             "s3:PutObject",
             "s3:AbortMultipartUpload",
-            "s3:ListMultipartUploadParts",
-            "s3:GetBucketVersioning",
+            "s3:ListMultipartUploadParts"
+         ],
+         "Effect": "Allow",
+         "Resource": ["arn:aws:s3:::<STORAGE_NAMESPACE>/*"],
+         "Principal": {
+            "AWS": ["arn:aws:iam::<ACCOUNT_ID>:role/<IAM_ROLE>"]
+         }
+      },
+      {
+         "Sid": "<STMT_ID2>",
+         "Action": [
             "s3:ListBucket",
             "s3:GetBucketLocation",
-            "s3:ListBucketMultipartUploads",
-            "s3:ListBucketVersions"
+            "s3:ListBucketMultipartUploads"
          ],
          "Effect": "Allow",
-         "Resource": ["arn:aws:s3:::<BUCKET_NAME>", "arn:aws:s3:::<BUCKET_NAME_WITH_PATH_PREFIX>/*"],
+         "Resource": ["arn:aws:s3:::<BUCKET>"],
          "Principal": {
-           "AWS": ["arn:aws:iam::<ACCOUNT_ID>:role/<IAM_ROLE>"]
+            "AWS": ["arn:aws:iam::<ACCOUNT_ID>:role/<IAM_ROLE>"]
          }
-       }
-     ]
-   }
-   ```
+      }
+   ]
+}
+```
+* Replace `<STORAGE_NAMESPACE>` with your [storage namespace](../../understand/model.md#concepts-unique-to-lakefs) which will be in the form of `<BUCKET_NAME>/<PATH>` (`PATH` can be empty).
+* Replace `<ACCOUNT_ID>` and `<IAM_ROLE>` with values relevant to your environment.
+* Replace `<BUCKET>` with the **bucket** that lakeFS is managing.
+* Replace `<STMT_ID1>`, `<STMT_ID2>` and `<POLICY_ID>` with string IDs of your choice.
+* `IAM_ROLE` should be the role assumed by your lakeFS installation.
+
+Alternatively, if you use an AWS user's key-pair to authenticate lakeFS to AWS, change the policy's Principal to be the user:
+
+```json
+ "Principal": {
+   "AWS": ["arn:aws:iam::<ACCOUNT_ID>:user/<IAM_USER>"]
+ }
+```
 
-   Replace `<ACCOUNT_ID>`, `<BUCKET_NAME>` and `<IAM_ROLE>` with values relevant to your environment.
-   `IAM_ROLE` should be the role assumed by your lakeFS installation.
+## Metadata Permissions Policy (minimal permissions)
 
-   Alternatively, if you use an AWS user's key-pair to authenticate lakeFS to AWS, change the policy's Principal to be the user:
+lakeFS requires permissions to access the `_lakefs` prefix under your storage namespace, in which the metadata
+objects are stored ([learn more](../../understand/versioning-internals.md#constructing-a-consistent-view-of-the-keyspace-ie-a-commit)).  
+By setting this policy you'll be able to perform only metadata operations through lakeFS, meaning that you'll **not** be able
```

Comment:
Can we emphasize this even more?  I worry that people will try "minimal permissions" (say, because they're easier to ask from your security team), but then run into difficulties.

In particular, given that the section ends "create your first lakeFS repository", giving tied-down permissions hints that the audience might not be a good match for minimal permissions.